types/V1.mli

```diff
@@ -51,8 +51,8 @@ module type TIME = sig
   type +'a io
   (** The type for potentially blocking I/O operation *)
 
-  val sleep: float -> unit io
-  (** [sleep nsec] Block the current thread for. {b FIXME:} remove float. *)
+  val sleep: int64 -> unit io
+  (** [sleep n] Block the current thread for n nanoseconds. *)
```

Comment:
now raises in 70fc815 (implementation updated in https://github.com/hannesm/mirage-platform/tree/sleep-nsec)